gpkit/constraints/tight.py

```diff
@@ -17,9 +17,9 @@ def process_result(self, result):
         for constraint in self.flat():
             if isinstance(constraint, (PosynomialInequality,
                                        SignomialInequality)):
-                leftsubbed = constraint.left.sub(variables)
-                rightsubbed = constraint.right.sub(variables)
-                rel_diff = abs(1-(leftsubbed/rightsubbed).value)
+                leftsubbed = constraint.left.sub(variables).value
+                rightsubbed = constraint.right.sub(variables).value
+                rel_diff = abs(1 - leftsubbed/rightsubbed)
```

Comment:
The underlying issue is that (due to an unbounded variable), `leftsubbed` and `rightsubbed` are both `inf`.

The fix above results in a `RuntimeWarning: invalid value encountered in double_scalars` during `process_result`, which is then followed by another `RuntimeWarning` from `check_solution`.

Do we definitely want `process_result` to get called before `check_solution`? In the case of #592, it seems to me that the right behavior would be to show the user that the problem doesn't even solve -- before checking constraint tightness.